packages/mdc-textfield/icon/README.md

```diff
@@ -19,19 +19,17 @@ Icons describe the type of input a text field requires. They can also be interac
   </li>
 </ul>
 
-## Usage
+## Basic Usage
 
 ### HTML Structure
 
 ```html
 <i class="material-icons mdc-text-field__icon" tabindex="0" role="button">event</i>
 ```
 
-### Usage within `mdc-text-field`
+## Variants
 
-Leading and trailing icons can be applied to text fields styled as `mdc-text-field--box` or `mdc-text-field--outlined`. To add an icon, add the relevant class (either `mdc-text-field--with-leading-icon` or `mdc-text-field--with-trailing-icon`) to the root element, add an `i` element with your preferred icon, and give it a class of `mdc-text-field__icon`.
-
-#### Leading icon
+### Leading icon
```

Comment:
We should keep the paragraph that was here, perhaps before any of the subheadings? We at least can't lose the note about this only being applicable to box and outlined.